src/rgw/store/sfs/sqlite/versioned_object/versioned_object_conversions.cc

```diff
@@ -0,0 +1,53 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t
+// vim: ts=8 sw=2 smarttab ft=cpp
+/*
+ * Ceph - scalable distributed file system
+ * SFS SAL implementation
+ *
+ * Copyright (C) 2022 SUSE LLC
+ *
+ * This is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License version 2.1, as published by the Free Software
+ * Foundation. See file COPYING.
+ */
+#include "versioned_object_conversions.h"
+#include "../conversion_utils.h"
+
+namespace rgw::sal::sfs::sqlite  {
+
+ObjectState getObjectState(uint int_state) {
```

Comment:
could we rename this `int_state` to something more descriptive? Like `obj_state`?

Reply:
sure, I tried to name it int_ to reflect that the input value is a integer value from the db, not an enum, but I'll change it